BritneyS/GetFed
# Food entry

## What you did :question:
- Created Food Entry view to enable users to enter custom food entries
- Installed Core Data

## How you did it :white_check_mark:
- Used stack views to organize the UI labels and textfields for the Food Entry view
- Added scroll view (in anticipation of adding and Image view/other entry fields in the future)
- Set the keyboard type for the protein, carbs, and fat textfields to decimal pad (may revisit since this keyboard has no return button) **UPDATE: Decimal Pad changed to Number Pad**


## How to test it :microscope:
- Run the app
- Tap 'Food Entry'
- Note that the Food Entry view is visible
- Enter any values into the textfields (with numeric entries for protein, carbs, and fat)
- Note that the alphabetical keyboard appears when the 'Food Name' and 'Brand Name' textfields are selected, and the numeric keyboard appears when the 'Protein', 'Carbs', and 'Fat' textfields are highlighted (according to Simulator keyboard settings)
- Tap the Save button
- Note that the keyboard is dismissed from the screen and the entries print to the terminal
- Tap any textfield, and tap the Cancel button
- Note that any keyboard is dismissed, and the user is returned to the Home view


## Screenshots (if applicable) :camera:
![simulator screen shot - iphone xr - 2018-12-06 at 09 23 00](https://user-images.githubusercontent.com/8409475/49590754-a49d6d00-f93a-11e8-9a4d-ea6c12d8ea60.png)
![simulator screen shot - iphone xr - 2018-12-06 at 09 23 06](https://user-images.githubusercontent.com/8409475/49590762-a9fab780-f93a-11e8-9335-03c7fc44e13b.png)
![simulator screen shot - iphone xr - 2018-12-06 at 09 30 30](https://user-images.githubusercontent.com/8409475/49590771-af580200-f93a-11e8-929f-19337ebc7a2b.png)
![simulator screen shot - iphone xr - 2018-12-06 at 09 30 46](https://user-images.githubusercontent.com/8409475/49590783-b67f1000-f93a-11e8-8ae8-2fe7924e08a8.png)
![screen shot 2018-12-06 at 9 31 11 am](https://user-images.githubusercontent.com/8409475/49590794-bed74b00-f93a-11e8-8e21-e45b3a1ceaef.png)
##
**UPDATE: Decimal pad changed to Number Pad**
![simulator screen shot - iphone xr - 2018-12-06 at 15 28 35](https://user-images.githubusercontent.com/8409475/49611197-cdd7f080-f96e-11e8-9754-1ca0bc8ad5c9.png)
